File: microbenchmarks/prime_summing.py
def f():
    total = 0
    i = 1
    while i < 2000000:
        i = i + 1
        j = 2
        while j * j <= i:
            if i % j == 0:
                break
            j = j + 1
        else:
            total = total + i
    print total
f()

File: microbenchmarks/prime_summing.cpp
#include <cstdio>

int main(int argc, char** argv) {
    long long total = 0;
    for (long long i = 2; i < 2000000; i++) {
        bool prime = true;
        for (int j = 2; j * j <= i; j++) {
            if (i % j == 0) {
                prime = false;
                break;
            }
        }

        if (prime)
            total += i;
    }
    printf("%lld\n", total);
    return 0;
}

